src/extension.ts

```diff
@@ -102,11 +105,33 @@ export function activate(context: ExtensionContext) {
     settings['logs']['file'] = logfile;
     return settings;
   }
+}
 
-
-
-
+function verifyConfigurations() {
+  let configXML = workspace.getConfiguration();
+  let autoCloseTags = configXML.get("xml.completion.autoCloseTags");
+  let autoClosingBrackets = configXML.get("[xml]")["editor.autoClosingBrackets"];
+  if (autoCloseTags && autoClosingBrackets != "never") {
+    window.showInformationMessage(
+      "'xml.completion.autoCloseTags' cannot be enabled while '[xml].editor.autoClosingBrackets' does not equal 'never'.", 
+        "autoCloseTags = false", 
+        "autoClosingBrackets = 'never'").then((selection) => {
+          if(selection == "autoClosingBrackets = 'never'") {
+            workspace.getConfiguration().update("[xml]", { "editor.autoClosingBrackets": "never" }, true).then(
+              () => console.log('[xml].editor.autoClosingBrackets globally set to never'),
+              (error) => console.log(error)
+            );
+          }
+          else {
```

Comment:
be careful, this is the case when user presses escape. You don't want to do anything if action gets cancelled on purpose

Reply:
good catch, ill fix